contribs/src/main/java/com/netflix/conductor/contribs/lock/LocalOnlyLock.java

```diff
@@ -39,7 +41,9 @@ public Semaphore load(String key) {
                     return new Semaphore(1, true);
                 }
             };
-    private static final LoadingCache<String, Semaphore> CACHE =
+    private static final ConcurrentHashMap<String, ScheduledFuture<?>> scheduledFutures =
```

Comment:
Please use capitals for `scheduledFutures` and `lockIdtoSemaphoreMap` since they are constants.